src/test/java/uk/co/orchardsystems/cinema_ticketing/CinemaServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package uk.co.orchardsystems.cinema_ticketing;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.co.orchardsystems.cinema_ticketing.Exceptions.NoScreeningsFoundException;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = CinemaTicketingApplication.class)
+public class CinemaServiceTest {
```

Comment:
This looks good. I wonder if there's another way to structure the code and tests that means you weren't hardcoding the exact structure of the JSON in the test. 
i.e. if `rating` were changed to `ageRating`, would that be worth a failing test? Part of the point of NOSQL is that you can shove anything in the DB, e.g. non-standard fields.

Just thinking out loud.

Reply:
I think you can add new fields and it shouldnt complain (as the comparison of the JSON is set to LENIENT) but modifying the name of an expected field would cause problems.

I agree though there is a tight coupling between the database schema if you like and the application logic